src/layouts/Field/Field.md

```diff
@@ -108,3 +108,38 @@ To make sure that forms are accessible, Fields supports associating labels with
   </Field>
 </Field.Group>
 ```
+
+### Providing a horizon alignment
```

Comment:
It's good that we have a place that is testing this code. However, I would prefer we leave testing to the Storybook pages and keep the documentation clean. You could use it in the toggle in the first example on this page.